app/src/main/kotlin/com/wire/android/ui/calling/OngoingCallScreen.kt

```diff
@@ -75,7 +75,7 @@ private fun OngoingCallTopBar(
 }
 
 @Composable
-private fun CallingControls(ongoingOngoingCallViewModel: OngoingCallViewModel = hiltViewModel()) {
+private fun CallingControls(ongoingCallViewModel: OngoingCallViewModel = hiltViewModel()) {
```

Comment:
instead of making this CallingControls dependent on OnGoingCallViewModel, create a lambda like : onHangUpClicked : () -> Unit, that way when you use this CallingControls(  onHangUpClicked {  } ), you directly see what the actions this composable has instead of going inside the Composable

Reply:
I don't want to make OngoingCallContent as a God method, let's keep as it is 🙂